ab_tool/views/intervention_point_pages.py

```diff
@@ -152,12 +154,13 @@ def edit_intervention_point_common(request, intervention_point_id):
     course_id = get_lti_param(request, "custom_canvas_course_id")
     intervention_point = InterventionPoint.get_or_404_check_course(
             intervention_point_id, course_id)
-    name = post_param(request, "name")
+    name = validate_name(post_param(request, "name"))
     notes = post_param(request, "notes")
     intervention_point.update(name=name, notes=notes)
-    # InterventionPointUrl creation
     intervention_pointurls = [(k,v) for (k,v) in request.POST.iteritems() if STAGE_URL_TAG in k and v]
     for (k,v) in intervention_pointurls:
+        validate_url(v) # Validates URLs using backend rules before any InterventionPointUrl object creation
```

Comment:
The frontend validation should catch nearly all cases. The additional backend validation is for incorrect or malicious POST requests.